x-pack/plugins/reporting/server/routes/common/generate/request_handler.ts

```diff
@@ -170,6 +170,7 @@ export class RequestHandler {
   public async handleGenerateRequest(exportTypeId: string, jobParams: BaseParams) {
     const req = this.req;
     const reporting = this.reporting;
+    exportTypeId = exportTypeId.replace('?elasticInternalOrigin=true', '');
```

Comment:
Are we sure this is needed? It's possible we want to change the `path` variable in `x-pack/plugins/reporting/server/routes/internal/generate/generate_from_jobparams.ts` to work around this, if it's needed.

Reply:
No good call we don't need to do this 😌